### PR TITLE
Publish external lib examples

### PR DESCRIPTION
**Pull request by Naylin15**
### Changes description
Hi @ajsb85 

In this PR I add the external lib information

Thank you

### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [x] Docs have been added/updated

### Estimated time

|Assignee|:tomato:|
|:--|:--:|
|@Naylin15|1|

### References

Closes #N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,18 +63,18 @@ Soon to be published
 #### Gradle
 
 ```groovy
-compile 'org.glpi:api:1.0.0'
+compile 'org.glpi:api:0.1.0@aar'
```
Comment (ajsb85) [inline README.md]: Use version 1.0

File: README.md [hunk 2]
```diff
@@ -65,18 +63,18 @@ Soon to be published
...
 
 ```
-<dependency org='org.glpi' name='api' rev='0.1.0'>
+<dependency org='org.glpi' name='api' rev='1.0.0'>
   <artifact name='api' ext='pom' ></artifact>
 </dependency>
 ```
 
-You can also find us on [**Bintray repository**](https://bintray.com/flyve-mdm/). -->
+You can also find us on [**Bintray repository**](https://bintray.com/glpi-project/maven-repository/android-api-client).
```
Comment (ajsb85) [inline README.md]: You needs to deploy as `jar` and not `aar` @Ivans51 
Because we can use this lib for Android and any Java project.

**Review by ajsb85 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Naylin15) [reply README.md]: Changes made 👍

File: README.md (hunk 2 above)
Comment (ajsb85) [reply README.md]: @Naylin15 you can use the same slug of the repo for the slug on bintray `java-library-glpi`

Comment (Naylin15) [reply README.md]: Changed 👍

**Review by ajsb85 — APPROVED** [review]

**Pull request merged**